### PR TITLE
Jenkins node updates

### PR DESCRIPTION
**Pull request by ekohl**
* Drop MySQL from nodes
* Update RVM form 1.29.9 to 1.29.10
* Ordering fixes
* Experimental conditionals for EL8 nodes
* Vagrant config to allow spinning up nodes

### CODE REVIEW

File: puppet/modules/slave/manifests/init.pp [hunk 1]
```diff
@@ -229,10 +233,9 @@
   include slave::java
 
   # Databases
-  include slave::mysql, slave::postgresql
-  slave::db_config { 'mysql': }
+  include slave::mysql
```
Comment (ehelms) [inline puppet/modules/slave/manifests/init.pp:236]: This is still used by something?

Comment (ekohl) [reply puppet/modules/slave/manifests/init.pp:236]: If you notice the actual class you notice that it now ensures the package and db config are absent. That means after this has been rolled out we can remove it after a while. I assumed that a package removal stops the service. It doesn't remove the actual storage (DBs etc).

File: puppet/modules/slave/manifests/init.pp [hunk 2]
```diff
@@ -229,10 +233,9 @@
...
+  include slave::postgresql
   slave::db_config { 'sqlite3': }
```
Comment (evgeni) [inline puppet/modules/slave/manifests/init.pp:238]: I think this can be `ensure: absent` these days

Comment (ekohl) [reply puppet/modules/slave/manifests/init.pp:238]: SQLite is still used in the 2.0 jobs. https://github.com/theforeman/foreman-infra/pull/1497 would probably solve it, but that PR is slow to be merged.

Comment (evgeni) [reply puppet/modules/slave/manifests/init.pp:238]: it's merged now.

**Comment by ekohl** [comment]
Rebased

File: puppet/modules/slave/manifests/init.pp (hunk 2 above)
Comment (ekohl) [reply puppet/modules/slave/manifests/init.pp]: Added as a separate commit now.

File: puppet/modules/slave/manifests/init.pp (hunk 1 above)
Comment (evgeni) [reply puppet/modules/slave/manifests/init.pp:236]: add a "#TODO remove when all nodes have been cleaned"?

**Review by evgeni — APPROVED** [review]

**Pull request merged**